scripts/mods/Penlight Lua Libraries/pl/array2d.lua

```diff
@@ -15,7 +15,7 @@ local imap,tmap,reduce,keys,tmap2,tset,index_by = tablex.imap,tablex.map,tablex.
 local remove = table.remove
 local splitv,fprintf,assert_arg = utils.splitv,utils.fprintf,utils.assert_arg
 local byte = string.byte
-local stdout = io.stdout
+local stdout = function(...) end --io.stdout

```

Comment:
`stdout` is a file object, so this will crash in `array2d.write` when `:write()` is called.
Just a heads up. Might not be a problem if the function is never called.

```suggestion
local stdout = { write = function() end } --io.stdout
```